packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/connectors.html

```diff
@@ -0,0 +1,166 @@
+<div class="container-fluid px-0 mt-3" data-ui-view>
+    <!-- error messages -->
+    <cui-message-container
+        data-messages="$ctrl.messages"
+    ></cui-message-container>
+
+    <h2 data-translate="pci_databases_connectors_tab_title"></h2>
+
+    <oui-datagrid
+        class="mt-5"
+        data-page-size="10"
+        data-rows="$ctrl.connectorsList"
+    >
+        <oui-datagrid-column
+            data-title=":: 'pci_databases_connectors_name' | translate"
+            data-property="name"
+            data-sortable="asc"
+            data-type="string"
+            data-searchable
+            data-filterable
+        ></oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_databases_connectors_connector' | translate"
+            data-property="connectorInformation.name"
+            data-sortable="asc"
+            data-type="string"
+            data-searchable
+            data-filterable
+        >
+            <div class="d-flex align-items-center">
+                <span
+                    class="round-icon oui-icon mr-2 oui-background-p-400 oui-color-p-050"
+                    ng-class="{
+                    'oui-icon-arrow-down': $row.connectorInformation.type === $ctrl.CONNECTOR_TYPES.SINK,
+                    'oui-icon-arrow-up': $row.connectorInformation.type === $ctrl.CONNECTOR_TYPES.SOURCE
+                    }"
+                >
+                </span>
+                <div>
+                    <h6
+                        ng-bind="$row.connectorInformation.name"
+                        class="mb-0"
+                    ></h6>
+                    <span class="oui-paragraph">
+                        <span
```

Comment:
Nitpick: move this logic of building a string to a controller. Controller can use `` and its more efficient.

Reply:
I'm not sure how to do handle the documentation link ? 
```
getConnectorInfoString(row) {
    let connectorInfoString =  `Version: ${row.connectorInformation.version} | Author: ${row.connectorInformation.author}`;
    if (row.connectorInformation.documentationUrl) {
        // ?
    }
    return connectorInfoString;
}
```

Reply:
You can keep the URL part in template. Rest of the string construction can be moved to controller.